src/NuGet.Clients/PackageManagement.UI/PackageLoader.cs

```diff
@@ -485,7 +487,7 @@ private IEnumerable<string> GetSupportedFrameworks()
 
                 // if the package does not exist in the remote source, NuGet should
                 // try getting metadata from the local resource.
-                if (String.IsNullOrEmpty(metadata.Summary) && localResource != null)
+                if (String.IsNullOrEmpty(metadata?.Summary) && localResource != null)
```

Comment:
Is it possible for the package to just have an empty summary? 

Really I would expect local to be used first for installed packages.